server/db/models/product.js

```diff
@@ -0,0 +1,26 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Product = db.define('product', {
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false
+  },
+  description: {
+    type: Sequelize.TEXT,
+    allowNull: false
+  },
```

Comment:
@garrett-green the same as name, lets review what is the best practice, and then we can update either of our model.